microscopium/data/default-settings.yaml

```diff
@@ -0,0 +1,57 @@
+global:
+  # where to put the output file
+  output-dir: ''
+# The first phase is a feature computation from images specified in a csv file.
+# Input csv files should have one row per input image and minimally three
```

Comment:
Including replicate, you now have four columns, not three. Unless you mean to say here that the replicate column is optional, in which case can you make it clearer?